magma/magma
# refactor: remove job queue impl reindexer

## Summary
- remove reindexer's nonsingleton version
- rename reindexer's singleton version to be the default reindexer
- remove all dependencies correlated with the nonsingleton reindexer that is no longer being used



## Test Plan
- [x] run `orc8r` tests on intellij
- [x] deploy on dev env and validate it works
    - [x] deploy and see right services spun up
    - [x] validate manually 

**Test Steps:**
1. `./build.py` in `orc8r/docker`
2. `./run.py`
3.  Check that `orc8r_worker` is successfully running
     - `docker-compose exec controller bash`
     - `supervisorctl`
4. Exit out and make sure it's reindexing properly
     - `docker exec -it orc8r_postgres_1 psql -U magma_dev`
     - `select * from indexer_versions`
     - `update indexer_versions set version_actual=0;`
     - `select` should look like ` lte          |              0 |               1` to ` lte          |              1 |               1` within a minute

- [ ] deploy helm charts and validate it works
    - [ ] deploy and see right services spun up
    - [ ] validate reindexing manually 
   

## Additional Information
